org.freedesktop.Sdk.Extension.php73.json

```diff
@@ -0,0 +1,148 @@
+{
+	"id": "org.freedesktop.Sdk.Extension.php73",
```

Comment:
Does the minor version of PHP make much difference? I mean, if php 7.x are more-or-less compatible, maybe give the extension ID of `.php7`?

Reply:
In my experience, there is breakage between minor versions. I have 2 projects that work in pho 7.2 that don't in 7.3